Gemfile

```diff
@@ -7,12 +7,12 @@ gem 'addressable', '~> 2.3'
 gem 'coder'
 gem 'jemalloc'
 gem 'metriks', '0.9.9.6'
-gem 'metriks-librato_metrics', github: 'eric/metriks-librato_metrics'
+gem 'metriks-librato_metrics', git: 'https://github.com/eric/metriks-librato_metrics'
```

Comment:
That's a clone over the SSH protocol -- which would also work as an alternative to `https`. However, that requires SSH to be setup and associated with a GitHub account.